Jellyfin.Networking/Manager/INetworkManager.cs

```diff
@@ -93,7 +93,7 @@ public interface INetworkManager
         /// <summary>
         /// Retrieves the bind address to use in system url's. (Server Discovery, PlayTo, LiveTV, SystemInfo)
         /// If no bind addresses are specified, an internal interface address is selected.
-        /// (See above).
+        /// (See GetBindInterface(IPObject souce, out int? port)).
```

Comment:
All your "see above" should be changed to something like this
```suggestion
        /// (See <see cref="GetBindInterface(IPObject, out int?)"/>.
```